server/export/export.go

```
@@ -201,117 +238,214 @@ func (e *Export) generateXlsx(projects []types.Project, services []types.Functio
 
 	allServiceIndicatorMap := getServiceIndicatorMap(projects, servicesMapSortedKeys, servicesMap, projectToUsageIndicators)
 
-	// Number of columns by service
-	const nbColsService = 6
+	for _, project := range projects {
+
+		if project.Name == "" {
+			log.WithField("project.Name", project.Name).Error(fmt.Sprintf("The project %v has no name", project.Name))
+			continue
+		} else {
```

Eviter les else en go quand ont fait des préassertions 'fail fast'. Ici, on fait une assertion pour vérifier la présence du nom, et on passe à l'itération suivante.

```go
if project.Name == "" {
	log.WithField("project.Name", project.Name).Error(fmt.Sprintf("The project %v has no name", project.Name))
	continue
} else {
 // qqch
}
```
=> 

```go
if project.Name == "" {
	log.WithField("project.Name", project.Name).Error(fmt.Sprintf("The project %v has no name", project.Name))
	continue
}
//qqch
```